miniscope_io/cli/stream.py

```diff
@@ -43,7 +48,7 @@ def _capture_options(fn: Callable) -> Callable:
         type=(str, Any),
     )(fn)
     fn = click.option("--no-display", is_flag=True, help="Don't show video in real time")(fn)
-    fn = click.option("-b", "--binary", help="Path (.bin) to save raw binary output to")(fn)
+    fn = click.option("-b", "--binary_export", is_flag=True, help="Save binary to a .bin file")(fn)
```

Comment:
Yes, there is in development, and it might be yes for others. It's annoying to convert binary to video later in dev for long clips. Also, real-time is not necessary for rescuing partially corrupted buffers for users, so it might be an option to save the video and see if some improvement is possible later using the binary.